Packages/com.unity.multiplayer.samples.coop/package.json

```diff
@@ -7,9 +7,9 @@
   "unity": "2020.3",
   "description": "Utilities package built on top of Netcode for GameObjects, providing useful scripts and tools.",
   "dependencies": {
-    "com.unity.learn.iet-framework": "1.2.1",
-    "com.unity.multiplayer.tools": "1.1.0",
-    "com.unity.netcode.gameobjects": "1.4.0",
-    "com.unity.services.relay": "1.0.3"
+    "com.unity.learn.iet-framework": "3.1.3",
+    "com.unity.multiplayer.tools": "2.0.0-pre.3",
```

Comment:
is the pre-release version required for this package, or can it stay with 1.1.0?

Reply:
It's not required, no. Are there risks to upgrading Tools-side?

Reply:
I don't think so, apart from it being in pre-release

Reply:
For the sake of getting this PR in, it's been reverted. I've also asked Product to take a look!